deepchecks/tabular/checks/data_integrity/string_length_out_of_bounds.py

```diff
@@ -249,19 +247,17 @@ def compare_outlier_ratio(result: Dict):
             not_passing_columns = {}
             for column_name in result.keys():
                 column = result[column_name]
-                total_outliers = 0
-                for outlier in column['outliers']:
-                    total_outliers += outlier['n_samples']
-
+                total_outliers = sum((outlier['n_samples'] for outlier in column['outliers']))
                 ratio = total_outliers / column['n_samples']
                 if ratio > max_ratio:
                     not_passing_columns[column_name] = format_percent(ratio)
             if not_passing_columns:
-                return ConditionResult(ConditionCategory.WARN,
-                                       f'Found columns with outliers ratio above threshold: '
-                                       f'{not_passing_columns}')
+                details = f'Found {len(not_passing_columns)} columns with outliers ratio above threshold out of ' \
+                          f'{len(result)} columns: {not_passing_columns}'
+                return ConditionResult(ConditionCategory.WARN, details)
             else:
-                return ConditionResult(ConditionCategory.PASS)
+                details = f'Passed for {len(result)} columns'
```

Comment:
Why not give any data?

Reply:
if there are no outliers, there isn't any data to give